benchopt/cli/main.py

```diff
@@ -29,10 +30,17 @@ def get_datasets(ctx, args, incomplete):
     return [d for d in datasets if incomplete.lower() in d]
 
 
+def get_benchmark(ctx, args, incomplete):
+    skip_import()
+    benchmarks = [b.path for b in os.scandir('.') if b.is_dir()]
+    return [os.curdir] + [b for b in benchmarks if incomplete in b]
```

Comment:
I would check if the folder contains an objective.py file to only show the folders that are likely to be benchmark folders

Reply:
Done in e4f8f3b